README.md

```diff
@@ -99,6 +99,87 @@ Add this to your `.pre-commit-config.yaml`
       use `args: [--markdown-linebreak-ext=md]` (or other extensions used
       by your markdownfiles).  If for some reason you want to treat all files
       as markdown, use `--markdown-linebreak-ext=*`.
+- `check-xml-encoding` - Checks that xml files have required encoding as first line.
+    - To specify desired encoding use `args: [--encoding]`. Example:
+    `args: [
+            --encoding, '<?xml version="1.0" encoding="utf-8"?>',
+          ]`
```

Comment:
doesn't `check-xml` do the same? or is it explicitly making sure you're using `utf-8` encoding?